### ISSUE TITLE
CSS Modules don't work in tests

### ISSUE THREAD

**Issue by bartaz** [canonical-ols/javan-rhino#25]
Found it while working on #24 

Importing CSS module in JS causes tests (`npm test`) to fail:

```
SyntaxError: /Users/bartaz/Canonical/javan-rhino/src/components/navlink.css: Unexpected token (1:0)
> 1 | .link {
    | ^
  2 |   color: $ubuntu-orange;
  3 |   display: inline-block;
  4 |   padding: 5px;
```

It seems we need to provide CSS modules loader properly to test environment.

**Comment by bartaz** [comment]
There is a project https://github.com/istarkov/babel-plugin-webpack-loaders that might help solve the issue (as we already use babel register in tests)

**Comment by bartaz** [comment]
Fixed by #24
